src/sidebars/favourites/favourite-preview.html

```diff
@@ -12,7 +13,10 @@ <h4 class="pull-left"><strong>{{stock.code | uppercase}}</strong></h4>
                 </div>
             </div>
             <div class="row">
-                <div class="col-xs-2 col-xs-offset-10">
+                <div class="col-xs-9" style="padding-left: 0px">
+                    <svg id="{{stock.code}}chart" width="100%" height="65px" ng-init="renderChart(stock)"></svg>
+                </div>
```

Comment:
I think we are good. We are not putting extra logic in template and we need to execute something the moment this svg becomes available. The other solution would be to make a directive here, but directive inside directive may be a small overkill for now. If the logic for this grows, we could think about it.